files/default/sql/ddl/updates/1.3.0.sql

```diff
@@ -23,3 +23,6 @@ CREATE TABLE IF NOT EXISTS `feature_store_tag` (
       PRIMARY KEY (`id`),
       UNIQUE KEY `name_UNIQUE` (`name`)
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+ALTER TABLE `hopsworks`.`host_services` CHANGE COLUMN `service` `name` varchar(48) COLLATE latin1_general_cs NOT NULL;
```

Comment:
did you run it manually? also for the undo?

Reply:
yes